app/controllers/welcome_controller.rb

```diff
@@ -1,8 +1,8 @@
 class WelcomeController < ApplicationController
 
   def index
-    @project = Project.find_by({id: 1})
-    @available_sprokets = Sproket.where(project_id: "")
+    @project = Project.take
+    @available_sprokets = Sproket.where("project_id is null")
```

Comment:
Using the "project_id is null" does work, but I have a feeling that ActiveRecord has a better way to do this -- I noticed that the empty string "" did not match the database null.